vm/devices/pci/pcie/src/lib.rs

```diff
@@ -17,6 +19,8 @@ const PAGE_SHIFT: u32 = PAGE_SIZE.trailing_zeros();
 
 const VENDOR_ID: u16 = 0x1414;
 const ROOT_PORT_DEVICE_ID: u16 = 0xC030;
+const UPSTREAM_SWITCH_PORT_DEVICE_ID: u16 = 0xC031;
+const DOWNSTREAM_SWITCH_PORT_DEVICE_ID: u16 = 0xC032;
```

Comment:
Are these in a standard, or are these specific to OpenVMM? Either way, please add a comment. (The `VENDOR_ID` is more obviously Microsoft's)